File: 1844-replace-all-digits-with-characters/1844-replace-all-digits-with-characters.java
class Solution {
    public String replaceDigits(String s) 
    
    {
       
        for(int i=1;i<s.length();i+=2)
        {
            int val=s.charAt(i-1)+(s.charAt(i)-'0');
            char x=(char)val;
            s=s.substring(0,i)+(x)+s.substring(i+1);
            
        }
        return s;
        
    }
}

File: 1844-replace-all-digits-with-characters/1844-replace-all-digits-with-characters.cpp
class Solution {
public:
    string replaceDigits(string s)
    {
        for(int i=1;i<s.size();i+=2)
        {
            int val=(s[i-1])+s[i]-'0';
            s[i]=(char)val;
        }
        return s;
        
        
    }
};